tests/PHPStan/Command/AnalyseCommandTest.php

```diff
@@ -29,6 +29,48 @@ public function testConfigurationAutoDiscovery(string $dir, string $file): void
 		}
 	}
 
+	public function testInvalidAutoloadFile(): void
+	{
+		$autoloadFile = 'phpstan.123456789.php';
+		$dir = realpath(__DIR__ . '/../../../');
+
+		$originalDir = getcwd();
+		if ($originalDir === false) {
+			throw new \PHPStan\ShouldNotHappenException();
+		}
+		chdir($dir);
+
+		try {
+			$output = $this->runCommand(1, ['--autoload-file' => $autoloadFile]);
+			$this->assertEquals(sprintf('Autoload file "%s" not found.' . PHP_EOL, $dir . DIRECTORY_SEPARATOR . $autoloadFile), $output);
+		} catch (\Throwable $e) {
+			chdir($originalDir);
+			throw $e;
+		}
+	}
+
+	public function testValidAutoloadFile(): void
+	{
+		$autoloadFile = 'data/autoload-file.php';
+		$dir = __DIR__;
+
+		$originalDir = getcwd();
+		if ($originalDir === false) {
+			throw new \PHPStan\ShouldNotHappenException();
+		}
+		chdir($dir);
```

Comment:
Please don't call this in the test. You also forgot to reset the dir in case the test passes.

Just use absolute path in `$autoloadFile` instead.